src/trailer.c

```diff
@@ -42,23 +42,22 @@ static const char *next_line(const char *str)
 }
 
 /*
- * Return the position of the start of the last line. If len is 0, return -1.
+ * Return the position of the start of the last line. If len is 0, return 0.
```

Comment:
We cannot discern text with zero length and text with a single line only, now, can we? Both would return `0`. I think it'd be cleaner to just use an out parameter here

Reply:
Agreed; fixed.